packages/react-sdk/src/components/index.ts

```diff
@@ -157,56 +140,12 @@ export const registerComponentPropsMetas = (
   currentPropsMetas = undefined;
 };
 
-const defaultComponents = {
-  Box,
-  Body,
-  TextBlock,
-  Heading,
-  Paragraph,
-  Link,
-  RichTextLink,
-  Span,
-  Bold,
-  Italic,
-  Superscript,
-  Subscript,
-  Button,
-  Input,
-  Form,
-  Image,
-  Blockquote,
-  List,
-  ListItem,
-} as const;
-
-export type ComponentName = keyof typeof defaultComponents;
 type RegisteredComponents = Partial<{
   // eslint-disable-next-line @typescript-eslint/ban-types
   [name in ComponentName]: {};
 }>;
 
-let registeredComponents: RegisteredComponents | null = null;
-
-const componentNames = Object.keys(defaultComponents) as ComponentName[];
-
-export const getComponentNames = (): ComponentName[] => {
-  const uniqueNames = new Set([
-    ...componentNames,
-    ...Object.keys(registeredComponents || {}),
-  ]);
-
-  return [...uniqueNames.values()] as ComponentName[];
-};
-
-export const getComponent = (
-  name: string
-): undefined | typeof defaultComponents[ComponentName] => {
-  return registeredComponents != null && name in registeredComponents
-    ? (registeredComponents[
-        name as ComponentName
-      ] as typeof defaultComponents[ComponentName])
-    : defaultComponents[name as ComponentName];
-};
+export let registeredComponents: RegisteredComponents | null = null;
```

Comment:
how about undefined )
```suggestion
export let registeredComponents: RegisteredComponents;
```